src/Pages/Login.js

```diff
@@ -50,15 +52,13 @@ const Login = () => {
         value={email}
         onChange={handleChange}
         onKeyPress={handleKeyPress}
-        variant="outlined"
+        variant="filled"
         sx={{width: '60%', '& .MuiInputLabel-root': {color: 'white'}, '& fieldset': {borderColor: 'white'}, '& input': {color: 'white'}}}
       />
-      <Button variant="contained" size="medium" sx={{width: '60%', marginTop: '10px'}} onClick={handleClick}>
+      <Button variant="contained" size="medium" sx={{width: '60%', marginTop: '30px'}} onClick={handleClick}>
```

Comment:
Please avoid using inline styles in the future, please use styled components for that purpose.

Reply:
I will try and make it in the next pr